app/templates/components/quick-filter.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="ui action input fluid ">
```

Comment:
Remove this div and instead add the classes to the component definition in `quick-filter.js` itself.

Ref: https://guides.emberjs.com/v2.13.0/components/customizing-a-components-element/